gui/wxpython/photo2image/ip2i_manager.py

```diff
@@ -421,22 +416,22 @@ def __init__(
         # """Make a POINTS file """
 
         try:
-            fc = open(self.file["camera"])
-            fc_count = 0
-            for line in fc:
-                fc_count += 1
-                if "NUM" in line:
-                    storeLine = fc_count
-                    numberOfFiducial = int(line.split()[-1])
-            dataFiducialX = []
-            dataFiducialY = []
-            fc = open(self.file["camera"])
-            fc_count = 0
-            for line in fc:
-                fc_count += 1
-                if fc_count > storeLine:
-                    dataFiducialX.append(line.split()[1])
-                    dataFiducialY.append(line.split()[2])
+            with open(self.file["camera"]) as fc:
+                fc_count = 0
+                for line in fc:
+                    fc_count += 1
+                    if "NUM" in line:
+                        storeLine = fc_count
+                        numberOfFiducial = int(line.split()[-1])
+                dataFiducialX = []
+                dataFiducialY = []
```

Comment:
If you end up unindenting the next fc opened file, bring this out of the previous context manager too.